slides/week1/week1.org

```diff
@@ -22,9 +22,15 @@
 
 * The Linux terminal
 - Powerful text-based interface for interacting with your computer
-- (Almost) everything is a file
-- Every file is part of one big tree
-- Lots of tiny tools you can chain together to do useful things
+- (Almost) everything is a file or a folder
+- It lets you navigate your folders and manipulate files quickly
+- There are lots of tiny tools and commands you can use to do useful things
+
+* ATTENTION WINDOWS USERS
+- Terminal is not the same as Command Prompt
+- We do not recommend using CMD as it has different commands than Linux
```

Comment:
They *cannot* use cmd

Reply:
I will change the phrasing of that line but I am interested in knowing why they can't. I would like to point out that I went through all of software training last year using command prompt